docs/guides/src/main/server/importExport.adoc

```diff
@@ -0,0 +1,101 @@
+
+You are also able to configure how users are going to be exported by setting the `--users <strategy>` option. The values available for this
+option are:
+
+* *different_files*: Users export into different files subject to the maximum number of users per file. This is the default value for this option.
+
+* *skip*: {project_name} skips exporting users.
+
+* *realm_file*:  Users export to the same file with the realm settings. The file is similar to "foo-realm.json" with realm data and users.
+
+* *same_file*:  Users export to the same file but different from the realm file. The result is similar to "foo-realm.json" with realm data and "foo-users.json" with users.
+
+If you are exporting users using the `different_files` strategy, you should also be able to set how many users per file you want by setting the `users-per-file` option. If not specified, the default value is `50`.
+
+<@kc.export parameters="--dir <dir> --users different_files --users-per-file 100"/>
+
+== Exporting a Realm to a File
+
+To export a realm to a file, you can use the `--file <file>` option.
+
+<@kc.export parameters="--file <file>"/>
+
+When exporting realms to a file, the server is going to use the same file to store the configuration for all the realms being exported.
+
+== Exporting a specific realm
+
+If you do not specify a specific realm to export, all realms are exported. To export a single realm, you can use the `--realm` option as follows:
+
+<@kc.export parameters="[--dir|--file] <path> --realm my-realm"/>
+
+== Importing a Realm from a Directory
+
+To import a realm, you can use the `import` command.
+
+<@kc.import parameters="--help"/>
+
+After exporting a realm to a directory, you can use the `--dir <dir>` option to import the realm back to the server as follows:
+
+<@kc.import parameters="--dir <dir>"/>
+
+When importing realms using the `import` command, you are able to set if existing realms should be skipped, or if they should be overridden with the new configuration. For that,
+you can set the `--override` option as follows:
+
+<@kc.import parameters="--dir <dir> --override false"/>
+
+By default, the `--override` option is set to `true` so that realms are always overridden with the new configuration.
+
+== Importing a Realm from a File
+
+To import a realm previously exported in a single file, you can use the `--file <file>` option as follows:
+
+<@kc.import parameters="--file <file>"/>
+
+== Importing a Realm during Startup
+
+You are also able to import realms when the server is starting by using the `--import-realm` option.
+
+<@kc.start parameters="--import-realm"/>
+
+When you set the `--import-realm` option, the server is going to try to import any realm configuration file from the `data/import` directory. Each file in this directory should
+contain a single realm configuration.
+
+If a realm already exists in the server, the import operation is skipped.
```

Comment:
Feels a bit weird for me that we have different defaults for this one as for the import command. import command overwrites, here realms are skipped. Why is that? Should we align?

Reply:
We are re-enabling the legacy behavior as this is what people are expecting now in terms of feature parity.

We can probably use the same import strategy in both commands in the future.

Reply:
works for me

Reply:
I wouldn't necessarily expect that the auto-import works the same way like it did in legacy. At least, it will be possible again, no matter if I need to specify override/skip options additionally.